### PR TITLE
Fixing small issues in the code (#2)

### PR DESCRIPTION
**Pull request by thopiekar**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,11 +3,10 @@
 
 from distutils.core import setup
 import py2exe
-import UM
-import UM.Qt
-import cura
+import UM    #@UnusedImport
```
Comment (Ghostkeeper) [inline setup.py]: This import is used down at line 27: `UM.__file__`

**Comment by Ghostkeeper** [comment]
I don't necessarily agree with the changes to WindowsRemovableDrivePlugin.py. Generally it's advisable to use `from ctypes import windll`-type importing rather than `import ctypes`. That's a bit of a subjective thing though so I'd merge this if it weren't for the one comment that I added to the change of setup.py. If you'll remove the comment you placed there then it's good to go as far as I'm concerned.

**Comment by thopiekar** [comment]
Ok, the idea behind using ctypes.windll instead of windll was to save reimporting it (would be available as ctypes.windll and windll).
I'll remove the comment soon. For a reason PyDev said it is unused, but I didn't check whether it is true.

**Comment by thopiekar** [comment]
Done :wink:

**Pull request merged**